content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,147 @@
+title: Empower your documentation with addons
+date: 2024-04-18
+description: Read the Docs Addons let documentation authors add features to their documentation such as analytics, version feedback, and search as you type to documentation in a modular way where authors have control.
+category: Changelog
+tags: addons, hosting
+authors: Manuel Kaufmann
+status: published
+image: /images/enable-beta-addons.jpg
+image_credit: Photo by <a href="https://unsplash.com/@impatrickt?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Patrick Tomasso</a> on <a href="https://unsplash.com/photos/open-book-lot-Oaqk7qqNh_c?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+
+In the last year, we have been focusing ourselves on providing a better experience for authors and readers of the documentation.
+Today, we are thrilled to introduce our latest feature,
+designed on top of users' feedback to build a better experience for them in our platform: **Read the Docs Addons**.
+
+Read the Docs Addons helps authors to keep their documentation updated by focusing on the most relevant pages,
+while simplifying the review process --and also helps readers to navigate the documentation in a simple way,
+always knowing the versions they are reading avoiding confusions.
```

Comment:
This feels like a pretty broad claim, and I don't really understand how addons helps here. Is this referring to analytics & the version warning? We should explain how we're providing these benefits.

Reply:
This paragraph refers to _all the addons_ as a whole actually 😄 

- "focusing on the most relevant pages" -> analytics
- "simplifying the review process" -> docdiff
- "navigate the documentation in a simple way" -> flyout
- "knowing the versions they are reading" -> notifications

Each of these concepts are explained on each of the following sections.